olm-catalog/serverless-operator/hack/update-manifests.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Are we calling this from anywhere? Should we add it to the `Makefile` so it's automatically taken into account?

Reply:
Upstream they were already updated, so I am suprised we have old files here.

Perhaps update the script to pull em from upstream?

Reply:
Sure, added it to the Makefile.

And yes this script will pull them from upstream :+1: